mentorship ios/Models/ProfileModel.swift

```diff
@@ -9,39 +9,6 @@ import Combine
 
 final class ProfileModel: ObservableObject {
 
-    // MARK: - Variables
-    var profileData = ProfileData(
-        id: 0,
-        name: "",
-        username: "",
-        email: "",
-        bio: "",
-        location: "",
-        occupation: "",
-        organization: "",
-        slackUsername: "",
-        skills: "",
-        interests: "",
-        needMentoring: false,
-        availableToMentor: false
-    )
-
-    // MARK: - Functions
-    func saveProfile(profile: ProfileData) {
-        guard let profileData = try? JSONEncoder().encode(profile) else {
-            return
-        }
-        UserDefaults.standard.set(profileData, forKey: UserDefaultsConstants.profile)
-    }
-
-    func getProfile() -> ProfileData {
-        let profileData = UserDefaults.standard.data(forKey: UserDefaultsConstants.profile)
-        guard let profile = try? JSONDecoder().decode(ProfileData.self, from: profileData!) else {
-            return self.profileData
-        }
-        return profile
-    }
-
     // MARK: - Structures
     struct ProfileData: Codable {
```

Comment:
MemberModel and ProfileModel have the same field structure. Can we reuse?

Reply:
Sure

Reply:
@ApheleiaS I found that the profileModel returning from backend have 24 attributes. We have only used 13 in profileModel (same as memberModel) in the app because currently some features are missing from backend/or are not needed for frontend, which may be completed in future. Example: "terms_and_conditions_checked", "is_admin", "resume_url", "photo_url"
ProfileModel is focused on our own profile which will help when we make Profile page in app whereas memberModel is focused on other members detail. Can you please suggest what to do, or can I leave it?